rclcpp/include/rclcpp/node.hpp

```diff
@@ -146,6 +148,24 @@ class Node
     FunctorT callback,
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
+  const std::vector<rcl_interfaces::SetParametersResult> set_parameters(
+    const std::vector<rcl_interfaces::Parameter> & parameters);
+
+  const rcl_interfaces::SetParametersResult set_parameters_atomically(
+    const std::vector<rcl_interfaces::Parameter> & parameters);
+
+  const std::vector<rclcpp::parameter::ParameterVariant> get_parameters(
+    const std::vector<std::string> & names);
```

Comment:
> Why return a const non-reference here? If it can't be a reference what is the purpose of const then?

Fixed.

> Can we make the getter methods const somehow?

Done.